scope_shared/scope/testing/fake_data/fixtures_fake_activity.py

```diff
@@ -1,20 +1,19 @@
 import datetime
+import faker
+import pytest
 import random
 from typing import Callable, List
 
-import faker
-import pytest
-import scope.database.collection_utils as collection_utils
 import scope.database.document_utils as document_utils
 import scope.database.format_utils as format_utils
 import scope.schema
-
 import scope.testing.fake_data.enums
 import scope.testing.fake_data.fake_utils as fake_utils
 
 
 # TODO: Get the generation verified.
 
+# TODO: James Comment 2/22:  I am not sure this is all correct / consistent
```

Comment:
This won't generate fake activity if `values` list is empty in `fake_values_inventory` dict.

Reply:
Right, that would be an expected state of a collection